requirements.txt

```diff
@@ -1,8 +1,8 @@
 openwisp-controller~=0.7.post1
-influxdb>=5.2,<5.3
-django-notifications-hq>=1.6,<1.7
-django-celery-email>=3.0.0,<3.1
-djangorestframework>=3.11,<3.12
+influxdb~=5.2
+django-notifications-hq~=1.6
+django-celery-email~=3.0.0
+djangorestframework~=3.11
```

Comment:
I also noticed this now. We can rely on `openwisp-utils[rest]`, which we made on purpose to centralize the DRF version there (so we can update it there to update it on all the modules).
I think we can leave `openwisp-utils[rest]` unpinned so the version used is the one dictated by `openwisp-controller`. Can you try that please?